### ISSUE TITLE
Hotkeys2 and MAUI Blazor Hybrid compatibility

### ISSUE THREAD

**Issue by tnglemongrass** [jsakamoto/Toolbelt.Blazor.HotKeys2#17]
I try to use the Hotkeys2 library in a MAUI Blazor Hybrid App with .NET 8, following the instructions from the Hotkeys2 README.

Maybe there is just something missing in the README instructions?

**Description:**

When using the HotKeys2 library in a Maui Blazor Hybrid App, an error/warning is displayed on startup. However, the app runs fine and hotkeys work as expected.

**Steps to Reproduce:**

- Create a new ".NET MAUI Blazor Hybrid App" with .NET 8.0 in Visual Studio 2022.
- Follow the instructions on https://github.com/jsakamoto/Toolbelt.Blazor.HotKeys2 to set up the HotKeys2 library.
- Run the app and observe the error/warning message in the console.

**Expected Behavior:**

There should be no errors or warnings on startup.

**Actual Behavior:**

The following error message is displayed on startup:

```
Toolbelt.Blazor.HotKeys2.HotKeys: Error: Could not find 'Toolbelt.Blazor.getProperty' ('Toolbelt' was undefined).
Error: Could not find 'Toolbelt.Blazor.getProperty' ('Toolbelt' was undefined).
```

**Additional Information:**

The app runs fine and hotkeys work as expected, despite the error message.

A minimal reproducible example is here: https://github.com/tnglemongrass/ToolbeltHotkeys2Demo/

**Comment by jsakamoto** [comment]
@tnglemongrass Thank you for letting me know about the problem. I must have overlooked the Blazor Hybrid scenario. As a workaround for a while, could you try to add one `<script>` tag in the `wwwroot/index.html` for the content, "_content/Toolbelt.Blazor.GetProperty.Script/Toolbelt.Blazor.GetProperty.Script.lib.module.js", like the following pull request?

https://github.com/tnglemongrass/ToolbeltHotkeys2Demo/pull/1

Actually, I've never tried that workaround on my side yet because I've been a bit busy these weeks. I'm sorry for that. Anyway, I hope the workaround above works well. (And I'd like to fix this problem someday.)

**Comment by tnglemongrass** [comment]
@jsakamoto Yes, including that script tag fixes it. Thanks for looking into it and thanks a lot for that workaround!

**Comment by fingers10** [comment]
@jsakamoto I was also having the same issue when trying to run ilovedotnet blazor wasm project via blazor hybrid project... Adding the script as mentioned solves the exception. 

You can use https://github.com/ILoveDotNet/ilovedotnet/tree/main/MAUI to test.

**Comment by jsakamoto** [comment]
[FYI]
This problem is a bug in the .NET SDK.

See also: https://github.com/dotnet/maui/issues/15234

Fortunately, I've verified the problem went away on the .NET 9 SDK Preview 5. So, please take the workaround of explicitly adding a script loading as below until the release of the .NET 9 SDK.

```html
<!-- wwwroot/index.html -->
<!-- 👇 Add this script tag before the script tag, which loads the "blazor.webview.js" JavaScript file. -->
<script src="_content/Toolbelt.Blazor.GetProperty.Script/Toolbelt.Blazor.GetProperty.Script.lib.module.js" type="module"></script>
```

**Comment by Alpaczyk** [comment]
Hey,
If someone still has this problem and solution from @jsakamoto does not work for you.

1. Copy this code to your site.js file `function beforeWebStart() { n() }
function beforeStart() { n() } const n = () => { const n = window.Toolbelt = window.Toolbelt || {}, t = n.Blazor = n.Blazor || {}; t.getProperty = n => n.replace(/^window\./, "").split(".").reduce((n, t) => typeof n == "undefined" || n === null ? n : n[t], window) };`

2. Call this function before you register your shortcuts
    `await JSRuntime.InvokeVoidAsync("beforeWebStart");`

At least it helped me
